pcdsdevices/epics_motor.py

```diff
@@ -456,3 +456,50 @@ class MotorDisabledError(Exception):
     Error that indicates that we are not allowed to move.
     """
     pass
+
+
+def Motor(prefix, **kwargs):
```

Comment:
At first I balked at this constructor, but upon closer inspection I see that there is only one positional arg to motors.